spec/service_models/miq_ae_service_miq_provision_spec.rb

```diff
@@ -110,13 +110,12 @@ def invoke_ae
   end
 
   it "#set_vm_notes" do
-    method   = "$evm.root['#{@ae_result_key}'] = $evm.root['miq_provision'].set_vm_notes"
+    method = "$evm.root['#{@ae_result_key}'] = $evm.root['miq_provision'].set_vm_notes('bite_me')"
```

Comment:
I'll assume you did this just to see if I was paying attention.  Let's try again.   If you need a suggestion for a string to pass please let me know.